pkg/hooks/structs/structs.go

```diff
@@ -1,28 +1,11 @@
 package structs
 
-// ConnID is a conversion of the following C-Struct into GO.
```

Comment:
Please don't remove comments of this code.

Reply:
I have same query as stated above reasoning the commend just mentions "GO-equivalent" of outdated struct.
Also, for this file I maintained the "// Socket Data Event ....." since it seemed necessary. If you still want me to retain those type of comments, I will be sure to update.